openapi3.yaml

```diff
@@ -118,11 +117,60 @@ paths:
               schema:
                 $ref: >-
                   ./Schema/ingestionTrigger/responses/ingestionTriggerResponses.yaml#/components/schemas/errorMessage
+  /ingestion/{jobId}/retry:
+    put:
+      operationId: retryLayer
+      tags:
+        - ingestion
+      summary: retry a failed ingestion job caused by validation errors
+      parameters:
+        - name: jobId
+          in: path
+          description: The id of the job to retry
+          required: true
+          schema:
+            type: string
+            format: uuid
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/SuccessfulIngestionResponse'
+        '400':
+          description: Bad request - Job status is not FAILED or COMPLETED, or no errors in validation task, or shapefile has not changed
```

Comment:
in general, do not mention specific task type like `validation` in the api as its handle all task types for the job retry